assets/js/fullSearch.js

```diff
@@ -128,14 +129,43 @@ var pagination = document.getElementById('pagination');
 
   function clearPrevResults() {
     entries = [];
-    suggestions.innerHTML = "";
-    pagination.innerHTML = "";
+    suggestions.innerHTML = '';
+    pagination.innerHTML = '';
+    suggestions.classList.remove('loader-visible');
+    suggestions.classList.remove('d-none');
+  }
+
+  function addLoader() {
+    const loader = document.createElement('div');
+    loader.classList.add('loader');
+    suggestions.classList.add('loader-visible');
+    const text = document.createElement('span');
```

Comment:
Why do we create a new element each time. Wouldn't it be easier if we show/hide the element?

Reply:
Made it to show/hide loader